.github/workflows/test-runner-pr-integration.yml

```diff
@@ -0,0 +1,103 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Test Runner PR Integration Tests
+
+on:
+  push:
+    branches: [ dev ]
```

Comment:
We don't allow direct pushes to dev, but that flow also runs when you merge a PR into dev to test the new head (also, you'll have a ✅  or ❌  icon next to the latests commit box at the top of the repo). Results show up on the Actions tab of the repo.
It seems like it could catch issues with unpublished changes but might be pretty low signal. I'm inclined to leave it in for consistency, but could take it out

Reply:
That makes sense. Fine to leave it then.